quinn-udp/src/unix.rs

```diff
@@ -66,6 +66,100 @@ impl Default for UdpSocketState {
     }
 }
 
+/// Fallback implementation of `sendmmsg` using `sendmsg`
```

Comment:
Style nit: we usually have more of a top-down ordering of items, so ideally we would move the added `sendmmsg_with_fallback()` directly below `send()`, with `sendmmsg_fallback` below that, and similar for  the receive side.